src/Book/Book.UnitTests/Application/BookAPITestData.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Book.Infrastructure;
+
+namespace Book.UnitTests.Application;
+
+public partial class BookAPITest
+{
+    public static TheoryData<Guid, BookContext> BookAPITestDataGetFaildBookById => new()
```

Comment:
Как на счёт чего-нибудь такого?

```suggestion
    public static TheoryData<Guid, BookContext> BookAPITestDataGetFaildBookById
        => new()
        {
            { Guid.NewGuid(), new InMemoryBookDB().context },
            { Guid.NewGuid(), new InMemoryBookDB().context },
            { Guid.NewGuid(), new InMemoryBookDB().context },
            { Guid.NewGuid(), new InMemoryBookDB().context }
        };
```